extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -156,21 +164,22 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
       }
       const elapsedTime = currTime - startTime;
       if (elapsedTime < DURATION_MS) {
+        const {x, y, zoom} = this.animateTo_;
         const easing = easeInOutQuad(elapsedTime / DURATION_MS);
         this.storeService_.dispatch(Action.SET_PANNING_MEDIA_STATE, {
-          x: startPos.x + (this.x_ - startPos.x) * easing,
-          y: startPos.y + (this.y_ - startPos.y) * easing,
-          zoom: startPos.zoom + (this.zoom_ - startPos.zoom) * easing,
+          [this.groupId_]: {
```

Comment:
Plz either change the Action to `ADD_*`, or make sure you merge this new data into the old state before setting it, so we get a more accurate action name.